source/data-formats/bson.txt

```diff
@@ -117,6 +117,32 @@ The following example reads the sample BSON document from ``file.bson``:
       
       {"address": {"street": "Pizza St", "zipcode": "10003"}, "coord": [-73.982419, 41.579505], "cuisine": "Pizza", "name": "Mongo's Pizza"}
 
+Work with Raw BSON Data
+-----------------------
+
+You might prefer to use raw BSON documents in certain cases, such as when you move a
+document between databases or collections, when you must write binary blobs to a
+disk, or when you want to bypass the performance overhead of converting to and from
+{+language+} dictionaries. You can use the ``RawBSONDocument`` class to represent raw BSON
+documents. To use ``RawBSONDocument`` objects with your collection, set the
+``document_class`` parameter of the ``MongoClient`` constructor to ``RawBSONDocument``.
+
+The following example fetches the sample BSON document from a collection and converts it
+to a ``RawBSONDocument``:
```

Comment:
I: this example doesn't seem to be "converting" anything. It seems to be retrieving a document **as** a raw BSON document as this is the document class specified when creating the client
S: fix this description to match the code example